capstone/src/main/webapp/script.js

```diff
@@ -427,55 +421,90 @@ function sizeUnitConversion(size){
     }
     return size.toString()+" Bytes";
 }
+function createListAPKNameElement(texts) {
+  const card = document.createElement("div");
+  card.className = "card bg-primary text-dark";
+  const cardBody = document.createElement("div");
+  cardBody.className = "card-header";
+  cardBody.innerText =("APK Name: " + texts);
+
+  card.appendChild(cardBody);
+  
+  return card;
+}
+
+function createListHeaderElement(texts) {
+  const card = document.createElement("div");
+  card.className = "card bg-secondary text-white";
+  const cardBody = document.createElement("div");
+  cardBody.className = "card-header";
+  cardBody.innerText = texts;
+
+  card.appendChild(cardBody);
+  
+  return card;
+}
+function createListBodyElement(texts) {
+  const card = document.createElement("div");
+  card.className = "card bg-light text-dark";
+  const cardBody = document.createElement("div");
+  cardBody.className = "card-header";
+  cardBody.innerText = texts;
+
+  card.appendChild(cardBody);
+  
+  return card;
+}
 
 // getDisplay makes a user aware of how much space each component
 // of the APK consumes.
 function getDisplay(list) {
   // list is an arraylist containing strings, so we have to
   // reference its elements to create HTML content
-  d3.select("#displayComponent").text(apkName);
   const contentListElement = document.getElementById("displayComponent");
   contentListElement.innerHTML = '';
     
   for (var i = 0; i < list.length; i++) {
         contentListElement.appendChild(
-        createListElement('Res '));
+        createListAPKNameElement(apkName));
```

Comment:
All of this feels like it should have been a seperate PR. Is there a way to split this up? This is solving a different problem than the PR above.